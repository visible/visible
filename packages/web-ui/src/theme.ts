export interface Theme {
  foreground: {
    normal: string;
    wash: string;
  };
  background: {
    normal: string;
    wash: string;
  };
  highlight: {
    normal: string;
  };
  border: {
    normal: string;
  };
}
