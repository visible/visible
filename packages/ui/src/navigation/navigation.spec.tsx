import { render } from '@testing-library/react';
import React from 'react';
import { ThemeProvider } from 'styled-components';

import { theme } from '../theme';
import { Nav, NavItem } from '.';

describe('Navigation', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={theme}>
        <Nav>
          <NavItem>test</NavItem>
        </Nav>
      </ThemeProvider>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
