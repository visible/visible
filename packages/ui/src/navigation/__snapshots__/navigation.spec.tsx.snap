// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0;
  padding: 0;
  list-style: none;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 18px;
  font-weight: bold;
}

.c1:last-child {
  margin-right: 0;
}

.c1 > a {
  display: block;
  box-sizing: border-box;
  padding: 12px 0px;
  -webkit-transition: 0.15s ease-out;
  transition: 0.15s ease-out;
  color: #333333;
}

.c1 > a:hover {
  color: #ffaa01;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c1 > a. .sc-AykKD {
  visibility: display;
}

<ul
  class="c0"
>
  <li
    class="c1"
  >
    test
  </li>
</ul>
`;
